include/soundanalyzer.h

```diff
@@ -288,9 +278,9 @@ class SoundAnalyzer : public AudioVariables
             if (M5.Mic.record((int16_t *)ptrSampleBuffer.get(), MAX_SAMPLES, SAMPLING_FREQUENCY, false))

```

Comment:
I believe the thing is that the M5 path works on the M5 devices and the I2S path doesn't? But it could be that's an outdated perception of the state of affairs. I think we'd need @davepl to clear that up.